modules/ROOT/pages/syntax/operators.adoc

```diff
@@ -563,6 +564,101 @@ RETURN 'neo' + '4j' AS result
 |===
 
 
+[[match-string-is-normalized]]
+=== Checking if a `STRING` `IS NORMALIZED`
+
+The `IS NORMALIZED` operator is used to check whether the given `STRING` is in the `NFC` Unicode normalization form:
+
+.Query
+[source, cypher]
+----
+RETURN "the \u212B char" IS NORMALIZED AS normalized
+----
+
+The given `STRING` contains a non-normalized Unicode character: `\u212B`, therefore `false` is returned.
+To normalize the `STRING`, it is possible to use the xref:functions/string.adoc#functions-normalize[normalize()] function.
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+|===
+| normalized
+| false
+2+|Rows: 1
+|===
+
+*Considerations:*
+
+|===
+
+| `IS NORMALIZED` returns `null` when used on a non-`STRING` value, e.g. `RETURN 1 IS NORMALIZED` returns `null`.
+
+|===
+
+
+[[match-string-is-not-normalized]]
+=== Checking if a `STRING` `IS NOT NORMALIZED`
+
+The `IS NOT NORMALIZED` operator is used to check whether the given `STRING` is not in the `NFC` Unicode normalization form:
+
+.Query
+[source, cypher]
+----
+RETURN "the \u212B char" IS NOT NORMALIZED AS notNormalized
+----
+
+The given `STRING` contains a non-normalized Unicode character: `\u212B`, therefore `true` is returned, as the string is not normalized.
+To normalize the `STRING`, it is possible to use the xref:functions/string.adoc#functions-normalize[normalize()] function.
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+|===
+| notNormalized
+| true
+2+|Rows: 1
+|===
```

Comment:
```suggestion
.Query
[source, cypher]
----
RETURN "the \u212B char" IS NOT NORMALIZED AS notNormalized
----

.Result
[role="queryresult",options="header,footer",cols="1*<m"]
|===
| notNormalized
| true
2+|Rows: 1
|===

Because the given `STRING` contains a non-normalized Unicode character (`\u212B`), and is not normalized, `true` is returned.

To normalize a `STRING`, use the xref:functions/string.adoc#functions-normalize[normalize()] function.

```